### PR TITLE
COM-823 Vagrant as service

### PR DESCRIPTION
**Pull request by lauravignoli**

### CODE REVIEW

File: windows/source/bin/Release/serviceConfig.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Bahmni>
+  <serviceSettings>
+    <VM_NAME>Bahmni</VM_NAME>
```
Comment (lauravignoli) [inline windows/source/bin/Release/serviceConfig.xml:4]: @rmrlangford Can you please update Bahmni to Bahmni-RPM?

File: windows/source/bin/Release/serviceConfig.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Bahmni>
+  <serviceSettings>
+    <VM_NAME>Bahmni</VM_NAME>
+    <LOGS_PATH>C:\Bahmni\Bahmni Service Logs</LOGS_PATH>
```
Comment (lauravignoli) [inline windows/source/bin/Release/serviceConfig.xml:5]: @rmrlangford Can you please update this path to C:\EMR\bahmni\bahmni-service-logs?

Do we need to create this folder manually?

File: windows/source/bin/Release/serviceConfig.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Bahmni>
+  <serviceSettings>
+    <VM_NAME>Bahmni</VM_NAME>
+    <LOGS_PATH>C:\Bahmni\Bahmni Service Logs</LOGS_PATH>
+    <TIMER_INTERVAL_MINS>2</TIMER_INTERVAL_MINS>
+    <EXECUTION_DIRECTORY>C:\Bahmni</EXECUTION_DIRECTORY>
```
Comment (lauravignoli) [inline windows/source/bin/Release/serviceConfig.xml:7]: @rmrlangford What is this directory?

File: windows/source/app.manifest [hunk 4]
```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (lauravignoli) [inline windows/source/app.manifest:1]: @rmrlangford Don't we need to delete this file?

File: windows/source/bin/x64/Release/serviceConfig.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Bahmni>
```
Comment (lauravignoli) [inline windows/source/bin/x64/Release/serviceConfig.xml:2]: @rmrlangford Why is there another copy of this file?

File: windows/source/serviceConfig.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (lauravignoli) [inline windows/source/serviceConfig.xml:1]: @rmrlangford Why is there another copy of this file?

File: windows/README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+1. At configured intervals, checks to see if the VM is running
+1. If the VM is not running, start the VM
+1. If the VM is running and the service is stopped (user action or when the hosting windows machine has been shutdown), the service will gracefully shutdown the VM
+
+The windows service is configured to automatically start whenever the windows server is booted up. The deafult interval for checking the status of the VM is 10mins.
+
+### MSI Configuration Steps
+Assuming that VirtualBox is already installed on a 64bit Windows Professional machine together with the Bahmni Vagrant CentOS instance, please follow these steps to install and configure the Bahmni Windows Service:
+
+1. Download and install the [Bahmni Service MSI](https://github.com/jembi/cameroon-bahmni-config/blob/COM-823/windows/setup/Bahmni%20Service.msi) on your windows server.
```
Comment (lauravignoli) [inline windows/README.md]: @rmrlangford We will upload the Bahmni Service.msi on the release page instead, as this branch will be deleted. Please use this path instead: https://github.com/jembi/cameroon-openmrs-module-bahmniapps/releases/download/v1.5.0/Bahmni.Service.msi

File: windows/source/app.manifest (hunk 4 above)
Comment (rmrlangford) [reply windows/source/app.manifest:1]: @lauravignoli , nope, keeping it :)

File: windows/source/bin/x64/Release/serviceConfig.xml (hunk 5 above)
Comment (rmrlangford) [reply windows/source/bin/x64/Release/serviceConfig.xml:2]: @lauravignoli visual Studio.Net "packages" the solution files like this.

File: windows/source/serviceConfig.xml (hunk 6 above)
Comment (rmrlangford) [reply windows/source/serviceConfig.xml:1]: @lauravignoli  visual Studio.Net "packages" the solution files like this.

File: windows/README.md (hunk 7 above)
Comment (rmrlangford) [reply windows/README.md]: @lauravignoli , done

**Pull request merged**